packages/npm-mongo/wrapper.js

```diff
@@ -1,2 +1,9 @@
 NpmModuleMongodb = Npm.require('mongodb');
-NpmModuleMongodbVersion = Npm.require('mongodb/package.json').version;
+
+// Hard-code the version number of our fork. When we un-fork, revert this
+// change! Otherwise, NpmModuleMongodbVersion will either get the value of
+// "2.2.31" or the fork's URL, depending on in what order npm processes the
+// install commands.
+// Previously:
+// NpmModuleMongodbVersion = Npm.require('mongodb/package.json').version;
+NpmModuleMongodbVersion = "2.2.31";
```

Comment:
There might be a more reliable way to get the semantic version from the `mongodb/package.json` file, but this works too.